ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml

```diff
@@ -141,6 +141,70 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-ovn,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
do we want these to always run or only when we think we are making changes that might break local gateway. On second thoughts, figuring that out may be tricky and so better to run these always?

Reply:
yeah, there is a way to run CI based on which files/folders are touched. at least I read about it, but I haven't worked on anything with it yet and it only makes sense if some jobs are clearly never relevant unless some specific set of files are touched. I'm not sure if that's the case here.

Reply:
OK, I think we can optimize further later.